.vsts-ci.yml

```diff
@@ -254,6 +254,9 @@ jobs:
       - script: |
           git clean -d -f -x
         displayName: 'Clean source directory'
+      # LGBM_SKIP_DASK_TESTS=true is set below only because running the tests under emulation is so slow...
+      # in theory, 'lightgbm.dask' should work without issue on aarch64 Linux systems.
+      # That could probably be removed as part of https://github.com/microsoft/LightGBM/issues/6788
```

Comment:
All of this `LGBM_SKIP_DASK_TESTS` machinery could be removed if #6843 is merged before this.